src/schemes/boundary/open_boundary/boundary_zones.jl

```diff
@@ -116,11 +116,11 @@ struct InFlow{NDIMS, IC, S, ZO, ZW, FD}
 
         if !isapprox(abs(dot_), 1.0, atol=1e-7)
             throw(ArgumentError("`flow_direction` is not normal to inflow plane"))
-        else
-            # Flip the normal vector to point in the opposite direction of `flow_direction`
-            spanning_set[:, 1] .*= -sign(dot_)
         end
 
+        # Flip the normal vector to point in the opposite direction of `flow_direction`
+        spanning_set[:, 1] .*= -sign(dot_)
```

Comment:
@LasNikas please confirm that this is correct.

Reply:
The if branch would result in a termination?

Reply:
Yes, basically nothing changes. As @svchb said, it will terminate if the condition is true.